WooCommerce/Classes/POS/Presentation/PointOfSalePaymentsTestView.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+import Combine
+
+// Required for making a dummy order
+import Yosemite
+import Fakes
+
+class PointOfSalePaymentsTestViewModel: ObservableObject {
+    let cardPresentPayments: CardPresentPayments
+    var cancellables = Set<AnyCancellable>()
```

Comment:
`cancellables` is not currently being used, in case that you see any odd behaviour with the expected subscription lifecycle

Reply:
Thanks